ExampleApp/Assets/OptimoveSdk/Optimove.cs

```diff
@@ -59,86 +59,91 @@ public static void Initialize()
 
         void Awake()
         {
-                Optimove.Shared = this;
+            Optimove.Shared = this;
 
-// #if UNITY_ANDROID
-//             PollPendingPush();
-// #endif
+#if UNITY_ANDROID
+            PollPendingPush();
+#endif
 
         }
 
         #endregion
 
         #region Helper Functions
-        private bool isValidString(string value)
+        private bool IsValidString(string value)
         {
             return value != null && !value.Equals("");
         }
+
+
         #endregion
 
         #region User Association
 
         public void SetUserId(string userId)
         {
-                if (!isValidString(userId)){
-                        Debug.LogError("Invalid user id");
-                        return;
-                }
+            if (!IsValidString(userId))
+            {
+                Debug.LogError("Invalid user id");
+                return;
+            }
 
-                #if UNITY_IOS
-                        OptimoveSetUserId(userId);
-                #elif UNITY_ANDROID
-                        //AndroidProxy.CallStatic("setUserId", userId);
+#if UNITY_IOS
+                            OptimoveSetUserId(userId);
+#elif UNITY_ANDROID
+            AndroidProxy.CallStatic("setUserId", userId);
```

Comment:
what's up with formatting here? `OptimoveSetUserId` above does not have same indentation as `ndroidProxy.CallStatic("setUserId"` . Same for many. Was it auto-formatted? I was giving 1 tab everywhere

Reply:
yes, I have auto formatting on in my visual studio

Reply:
in any case there is no way we should have 5 different indentations in the same function. I have updated to have 1 tab everywhere. If you auto-format in visual studio again -- it will likely change it, but at least it's going to be consistent